GP-Engine-team/GP_Engine
# Window appearing too early fix

## Description
This pull request answers [this bug](https://trello.com/c/5wTvQybp/114-the-editors-window-appears-before-being-fully-initialized).
When launching the editor, this is what first appears:
![](https://trello-attachments.s3.amazonaws.com/600695e08122e82f6ad86ec9/60795b322dc8284b45391d9c/479ce43c27cfcbb7f7089837f1fb6b28/image.png)

This pull request makes the editor's window appear only once its content has been fully initialized.

## How to test
Launch the editor, watch the initialization in the console, and check that the editor appears only once the initialization is done
